### ISSUE TITLE
Some input devices remain uninitialized (udev regression after #10603)

### ISSUE THREAD

**Issue by grawity** [systemd/systemd#10916]
**systemd version the issue has been seen with**
> v239-2239-gc4b69e990

**Used distribution**
 > Arch Linux

**Unexpected behaviour you saw**

On boot (almost every time), many input devices remain "uninitialized" by udev, and therefore skipped by libinput. Usually only the touchpad works, but the rest are missing from `sudo libinput list-devices`.

The device nodes are present, but `udevadm info` shows:

```
P: /devices/platform/i8042/serio0/input/input4/event4
N: input/event4
L: 0
E: DEVPATH=/devices/platform/i8042/serio0/input/input4/event4
E: DEVNAME=/dev/input/event4
E: MAJOR=13
E: MINOR=68
E: SUBSYSTEM=input
```

Whereas the expected output is:

```
P: /devices/platform/i8042/serio0/input/input4/event4
N: input/event4
L: 0
S: input/by-path/platform-i8042-serio-0-event-kbd
E: DEVPATH=/devices/platform/i8042/serio0/input/input4/event4
E: DEVNAME=/dev/input/event4
E: MAJOR=13
E: MINOR=68
E: SUBSYSTEM=input
E: USEC_INITIALIZED=7285005
E: ID_INPUT=1
E: ID_INPUT_KEY=1
E: ID_INPUT_KEYBOARD=1
E: ID_BUS=i8042
E: ID_SERIAL=noserial
E: ID_PATH=platform-i8042-serio-0
E: ID_PATH_TAG=platform-i8042-serio-0
E: LIBINPUT_DEVICE_GROUP=11/1/1:isa0060/serio0
E: DEVLINKS=/dev/input/by-path/platform-i8042-serio-0-event-kbd
E: TAGS=:power-switch:
```

**Steps to reproduce the problem**

Automatically happens roughly on 4 out of 5 boots; bisected to commit c4b69e9 (merged via 60d540f as part of #10603).

Currently running c52cff07 which has no problems even after ~10 reboots.

**Comment by grawity** [comment]
Oh, duplicate of #10754.
